org.shadered.SHADERed.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: org.shadered.SHADERed
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: shadered
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
+  - --device=all
+
+modules:
+  - name: assimp
+    buildsystem: cmake-ninja
+    cleanup:
+      - "*.hpp"
+      - "*.h"
+      - "*.inl"
+    config-opts:
+      - -DASSIMP_BUILD_ASSIMP_TOOLS:BOOL=NO
+      - -DASSIMP_BUILD_TESTS:BOOL=NO
+    sources:
+      - type: archive
+        url: https://github.com/assimp/assimp/archive/v4.1.0.tar.gz
+        sha256: 3520b1e9793b93a2ca3b797199e16f40d61762617e072f2d525fad70f9678a71
+
+  - shared-modules/glew/glew.json
+
+  - shared-modules/glu/glu-9.json
+
+  - name: glm
+    buildsystem: cmake-ninja
+    cleanup: ["*"]
+    sources:
+      - type: archive
+        url: https://github.com/g-truc/glm/releases/download/0.9.9.2/glm-0.9.9.2.zip
+        sha256: 209b5943d393925e1a6ecb6734e7507b8f6add25e72a605b25d0d0d382e64fd4
+
+  - name: SFML
+    buildsystem: cmake-ninja
+    cleanup:
+      - "*.hpp"
+      - "*.h"
+      - "*.inl"
+    config-opts:
+      - -DSFML_INSTALL_PKGCONFIG_FILES=TRUE
+    sources:
+      - type: archive
+        url: https://www.sfml-dev.org/files/SFML-2.5.1-sources.zip
+        sha256: bf1e0643acb92369b24572b703473af60bac82caf5af61e77c063b779471bb7f
+
+  - name: shadered
+    buildsystem: cmake-ninja
+    cleanup:
+      - "*.hpp"
+      - "*.h"
+    sources:
+      - type: git
+        url: https://github.com/dfranx/SHADERed.git
+        tag: v1.3
+        commit: 1b6ab1d5ed0ecaf15dc161bf0313c571bb61d953
+
+      # SDL error check causes the application to not spawn a window, so remove it.
+      - type: patch
+        path: shadered-remove-sdl-error-check.patch
+
+      # AppStream metadata was fixed after SHADERed 1.3's release, so download it from the repository.
+      - type: file
+        url: https://raw.githubusercontent.com/dfranx/SHADERed/a4e1c817cda96508775db9b7a6d2e09c80453c5d/Misc/Linux/org.shadered.SHADERed.appdata.xml
+        sha256: dda483cdaf4d0baf0292ec0f2d43e34e397cc73e3d0d306a544289b05a22ba79
+
+    post-install:
+      # The icon and `.desktop` file are already installed by CMake, but they need to be renamed.
+      - mv /app/share/applications/{shadered,$FLATPAK_ID}.desktop
+      - desktop-file-edit --set-icon=$FLATPAK_ID /app/share/applications/$FLATPAK_ID.desktop
+      - mv /app/share/icons/hicolor/64x64/apps/{shadered,$FLATPAK_ID}.png
+      - install -Dm644 $FLATPAK_ID.appdata.xml -t /app/share/appdata/
```

Comment:
I would prefer using rename-desktop/rename-appdata/rename-icon instead.

Reply:
Ì tried using those, but it didn't work for some reason. I'll get the error message once I get back home.

Reply:
It turns out I needed to use `rename-icon` instead of `rename-icon-file`, while specifying the file extension in `rename-desktop-file`.